src/test/Components/Blueprints/Blueprints.test.js

```diff
@@ -0,0 +1,46 @@
+import { screen } from '@testing-library/react';
+import { rest } from 'msw';
+
+import { IMAGE_BUILDER_API } from '../../../constants';
+import {
+  emptyGetBlueprints,
+  mockGetBlueprints,
+} from '../../fixtures/blueprints';
+import { server } from '../../mocks/server';
+import { renderWithReduxRouter } from '../../testUtils';
+
+jest.mock('@redhat-cloud-services/frontend-components/useChrome', () => ({
+  useChrome: () => ({
+    isBeta: () => false,
+    isProd: () => true,
+    getEnvironment: () => 'prod',
+  }),
+}));
+
+jest.mock('@unleash/proxy-client-react', () => ({
+  useUnleashContext: () => jest.fn(),
+  useFlag: jest.fn((flag) =>
+    flag === 'image-builder.new-wizard.enabled' ? true : false
+  ),
+}));
+
+describe('Blueprints', () => {
+  test('renders blueprints page', async () => {
+    renderWithReduxRouter('', {});
+    const blueprintName = mockGetBlueprints.data[0].name;
```

Comment:
I've been thinking a lot about how to handle this in the tests in general recently, and I think we should de-couple the tests from the data that we use to generate the mock API responses.

So here we expect the name to be 'Dark Chocolate'. Let's just do `await screen.findByText('Dark Chocolate');`.

I know that means some additional overhead in the code, but it means we test exactly what we expect to get. It also means that if you look at the tests while `npm run stage-beta:msw` is running as your local dev environment, you'll be able to quickly make a 1:1 correlation between what's in your browser and what is in the test code.

Reply:
@regexowl thoughts?

Reply:
doesn't it make sense to use one source of truth for fixtures?  both for testing and for mocking API?

Reply:
Try thinking about it like this: if we use `await screen.findByText('Dark Chocolate')`, we are effectively testing _both_ the React components *and* the mocking API.

If we use `await screen.findByText(mockGetBlueprints.data[0].name)` and `mockGetBlueprints.data[0].name` is not what we expect it to be for whatever reason, and the page is rendered incorrectly, our tests would still pass.

Is that a reasonable way of looking at things?

Reply:
The blueprint's name in the fixture is a constant, in the test we verify that it propagated well from the API to the component, so to me it looks a bit redundant declaring the same name again. but, if it works for you let it be :)

Reply:
It is redundant. Try to remember I come from a traditional engineering background and we like to have lots of redundancy in our safety systems. ;) 

So let's go ahead and use `await screen.findByText('Dark Chocolate')`.

Reply:
done